.editorconfig

```diff
@@ -67,7 +67,7 @@ ij_javascript_array_initializer_wrap = off
 ij_javascript_assignment_wrap = off
 ij_javascript_binary_operation_sign_on_next_line = false
 ij_javascript_binary_operation_wrap = off
-ij_javascript_blacklist_imports = rxjs/Rx,node_modules/**,**/node_modules/**,@angular/material,@angular/material/typings/**
+ij_javascript_blacklist_imports = rxjs/Rx, node_modules/**, **/node_modules/**, @angular/material, @angular/material/typings/**
```

Comment:
Changes to this file should probably be a separate PR (I assume IntelliJ did this on its own).

Reply:
I had to make changes to .editorconfig so that index.md passes the spotless check since they were inconsistent with each other. When I did this, IntelliJ did auto-change this file because it didn't match the configs defined in itself. I'm happy to move the whitespace changes to a different PR, but the changes to the markdown section seem to be related here... unless you want me to move all of that to a different PR and get it merged to main and then merged into this PR before this PR can be merged. I'm good with whichever

Reply:
Created PR for this - https://github.com/terraware/terraware-server/pull/2873

Reply:
This file should no longer show as modified once the other PR gets merged to main and I merge it into this one

Reply:
No longer in this PR!